CI/circle_parallel.sh

```diff
@@ -8,6 +8,10 @@ if [ "$NODE_INDEX" = "1" ]; then
   cp CI/pom.xml.circleci pom.xml
   java -version
   mvn --quiet verify -Psamples
+elif [ "$NODE_INDEX" = "2" ]; then
+  echo "Running node $NODE_INDEX to test ensure-up-to-date"
+  mvn clean package -DskipTests -Dmaven.javadoc.skip=true
```

Comment:
"mvn clean install" was run before running this script (https://github.com/OpenAPITools/openapi-generator/pull/944/files#diff-29944324a3cbf9f4bd0162dfe3975d88R69) so I don't think we need to do it again or I probably miss the reason behind.

Reply:
It is just a matter of having `modules/openapi-generator-cli/target/openapi-generator-cli.jar`. I have put here the command that is sufficient to produce it (with a clean to be sure that it gets updated).

Reply:
On shippable, there was already a `mvn clean install`, so what I did is just moving it before `ensure-up-to-date` with the initial commit:

https://github.com/OpenAPITools/openapi-generator/commit/00f176ad956c06fc6027b4b0e22d982b928552c6#diff-033d042b39454e8e066ab38fd89610a5

Reply:
👍